Doesn't work on windows

I compiled on windows 8.1 successfully with python 2.7 and visual studio community edition but got an error when running npm start. Attached the npm-debug file.

[npm-debug.txt](https://github.com/scality/S3/files/428060/npm-debug.txt)

I identified two issues

1. init.js throws an assertion if the directories for localData already exist
```
AssertionError: Error creating data files Error: EEXIST: file already exists, mkdir 'C:\Users\foo\S3\localData\0'
    at C:\Users\foo\S3\init.js:81:13
    at C:\Users\foo\S3\node_modules\async\lib\async.js:52:16
    at C:\Users\foo\S3\node_modules\async\lib\async.js:267:21
    at C:\Users\foo\S3\node_modules\async\lib\async.js:44:16
    at C:\Users\foo\S3\init.js:75:20
    at FSReqWrap.oncomplete (fs.js:82:15)
```
as we are not using the windows error number -4075 (File already exists error) to skip the error. We should opt for using error code `EEXIST` instead of numbers for system error detection as they are consistent across the systems.

2. index.js - uses a list of files for babel to ignore, i.e, not to transpile. This is not working on Windows for reasons I don't know yet. I am working on a solution to statically transpile utapi module so that we don't have to provide this list.

I will be making some PRs to fix these issues.

--- a/init.js
+++ b/init.js
@@ -71,7 +71,7 @@ const subDirs = Array.from({ length: constants.folderHash },
 async.eachSeries(subDirs, (subDirName, next) => {
     fs.mkdir(`${dataPath}/${subDirName}`, err => {
         // If already exists, move on
-        if (err && err.errno !== -17) {
+        if (err && err.code !== 'EEXIST') {
             return next(err);
         }
         return next();

